app/instance-initializers/moment.js

```diff
@@ -0,0 +1,18 @@
+/* global chrome */
+
+function hasChromeI18N() {
+  return (typeof chrome === 'object') &&
+    chrome.i18n &&
+    chrome.i18n.getMessage;
+}
+
+export function initialize(appInstance) {
+  const moment = appInstance.lookup('service:moment');
+  const lang = hasChromeI18N() ? chrome.i18n.getMessage('lang_code') : 'en';
```

Comment:
initializer is still being called before the route, so this is not a solution. Please move moment initialization into application route